packages/carbon-web-components/src/components/ui-shell/header-menu-button.ts

```diff
@@ -77,10 +77,10 @@ class BXHeaderMenuButton extends FocusMixin(LitElement) {
   disabled = false;
 
   /**
-   * Usage mode of the side nav.
+   * If `true` will style the side nav to sit below the header
    */
-  @property({ reflect: true, attribute: 'usage-mode' })
-  usageMode = SIDE_NAV_USAGE_MODE.REGULAR;
+  @property({ type: String, reflect: true, attribute: 'is-child-of-header' })
+  isChildOfHeader = 'true';
```

Comment:
```suggestion
  @property({ type: Boolean, reflect: true, attribute: 'is-child-of-header' })
  isChildOfHeader = 'true;
```
```suggestion
  isChildOfHeader = 'true';
  
```
Maybe this can be a boolean instead?

Reply:
With some of the properties I specifically set it to accept string values because a lot of our adopters are using the components within a html file and so they aren't passing in actual boolean values. So I decided to make it accept string "true"/ "false" values instead to make it easier for adopters.

Reply:
Maybe it's worth it actually to just set a `is-not-child-of-header` property instead? so that all they have to do is set the property and not worry about the type of value to pass in.

`<cds-header-menu-button is-not-child-of-header ....>